### PR TITLE
Convert Prisma fields to singular and camelCase

### PR DESCRIPTION
**Pull request by feng-eric**
### Notion ticket link

[Change prisma model field](https://www.notion.so/uwblueprintexecs/086c1b564dbf478faa649a8880e34b81?v=9d3f8002a0d1477ba395616dfc082af7&p=e63074c4be274a53b7b5391cf6f7595d)

### Implementation description

- Changed fields of prisma model to be singular and camelCase
- Updated existing endpoints so change to prisma doesn't break endpoints

### Steps to test

1.

### What should reviewers focus on?

-

### Checklist

-   [ ] My PR name is descriptive and in imperative tense
-   [ ] I have run the linter
-   [ ] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -8,145 +8,145 @@ datasource db {
...
+  addressLine1      String            @map("address_line1")
+  addressLine2      String?           @map("address_line2")
+  postalCode        String            @map("postal_code") @db.VarChar(10)
+  cityName          String            @map("city_name") @db.VarChar(50)
+  province          province
+  preferredLanguage locale            @map("preferred_language")
+  createdAt         DateTime          @default(now()) @map("created_at") @db.Timestamptz(6)
+  updatedAt         DateTime          @default(now()) @map("updated_at") @db.Timestamptz(6)
+  user              User              @relation("parentsTousers", fields: [id], references: [id])
+  couponUser        CouponUsers[]     @relation("coupon_usersToparents")
```
Comment (feng-eric) [inline prisma/schema.prisma]: Not sure if fields with `[]` should be kept plural? I think they possibly should be?

File: prisma/schema.prisma [hunk 2]
```diff
@@ -8,145 +8,145 @@ datasource db {
...
-  users              User              @relation("parentsTousers", fields: [id], references: [id])
-  coupon_users       CouponUsers[]     @relation("coupon_usersToparents")
-  parent_of_students ParentOfStudent[] @relation("parent_of_studentsToparents")
-  parent_regs        ParentReg[]       @relation("parent_regsToparents")
-  waitlists          Waitlist[]        @relation("parentsTowaitlists")
+  id                Int               @id @default(autoincrement())
+  firstName         String            @map("first_name")
+  lastName          String            @map("last_name")
+  phoneNumber       String            @map("phone_number") @db.VarChar(50)
+  isLowIncome       Boolean?          @default(false) @map("is_low_income")
```
Comment (mahadzaryab1) [inline prisma/schema.prisma:26]: nit: can we have the mapping as the first decorator everywhere?

**Review by mahadzaryab1 — COMMENTED** [review]
Added a couple comments

**Review by mahadzaryab1 — APPROVED** [review]
LGTM

**Comment by feng-eric** [comment]
I'll let others merge stuff in first so they don't have to deal with merge conflicts.

**Review by jason17h — COMMENTED** [review]
For PRs that are currently open which interact with Prisma, should we change everything to camel case now or in a separate PR?

EDIT: 

> I'll let others merge stuff in first so they don't have to deal with merge conflicts.

Looks like they'll be in separate PRs after this one is merged.

**Review by jason17h — APPROVED** [review]

**Comment by feng-eric** [comment]
> For PRs that are currently open which interact with Prisma, should we change everything to camel case now or in a separate PR?
> 
> 
> 
> EDIT: 
> 
> 
> 
> > I'll let others merge stuff in first so they don't have to deal with merge conflicts.
> 
> 
> 
> Looks like they'll be in separate PRs after this one is merged.

Yeah, when I update this branch with staging after you guys merge your stuff in, I'll take the time to fix it

**Comment by jason17h** [comment]
Note that I am adding a `roles` field to the User schema in PR https://github.com/uwblueprint/social-diversity-for-children/pull/18 so that change will need to be added here as well. I kept it as plural for now just for consistency.

**Comment by feng-eric** [comment]
> Note that I am adding a `roles` field to the User schema in PR #18 so that change will need to be added here as well. I kept it as plural for now just for consistency.

Thanks for letting me know!

**Review by victoranyun — APPROVED** [review]
lgtm

**Comment by feng-eric** [comment]
> Note that I am adding a `roles` field to the User schema in PR #18 so that change will need to be added here as well. I kept it as plural for now just for consistency.

@jason17h I'm actually going to merge this in since it's blocking a few other PRs. We can make that change separately.

**Pull request merged**